src/bin-utils/get-script-by-prefix.js

```diff
@@ -0,0 +1,60 @@
+import prefixMatches from 'prefix-matches'
+import {keys, isPlainObject, isString, has} from 'lodash'
+
+/*
+  Converts a string or object with a "script" key into an object with
+   {
+    name,
+    script,
+    description
+   }
+*/
+export function scriptToObject(name, scriptArg) {
+  if (isString(scriptArg)) {
+    return {
+      name,
+      script: scriptArg,
+      description: '',
+    }
+  }
+  if (isPlainObject(scriptArg)) {
+    if (has(scriptArg, 'default')) {
+      return scriptToObject(`${name}.default`, scriptArg.default)
```

Comment:
In is validScript, it tests before `default` and `script`. How is that covered here? I am not sure  am seeing it.